examples/java-matter-controller/README.md

```diff
@@ -0,0 +1,67 @@
+# Matter Controller Java App Example
+
+This is a Matter Controller Java app that can be used to uses Matter to send
+messages to a Matter server.
+
+<hr>
+
+-   [Matter Controller Java App Example](#matter-controller-java-app-example)
+    -   [Requirements for building](#requirements-for-building)
+        -   [Gradle & JDK Version](#gradle--jdk-version)
+    -   [Preparing for build](#preparing-for-build)
+    -   [Building & Running the app](#building--running-the-app)
+
+<hr>
+
+<a name="requirements"></a>
+
+## Requirements for building
+
+You need Android SDK 21 & NDK downloaded to your machine. Set the
```

Comment:
Is this permanent? Why would we not just use the java SDK without android?

Reply:
Since we can not build Matter SDK with JDK now, to build java-matter-controller with JDK, we first need to be able to build Matter SDK with JDK to produce a compatible matter library. 
 https://github.com/project-chip/connectedhomeip/issues/23133.

Reply:
Since we can link CHIPController.jar and run the Java application built with android binary x84 directly on Linux, we plan to continue this route at this moment until we hit the problem